spacy/vectors.pyx

```diff
@@ -511,6 +511,9 @@ cdef class Vectors:
                     for i in range(len(queries)) ], dtype="uint64")
         return (keys, best_rows, scores)
 
+    def to_ops(self, ops: Ops):
```

Comment:
Perhaps we can document this one at https://spacy.io/api/vectors ?

Reply:
I'm not sure how often it would come up that a user would need to call this directly though.

Reply:
I don't know. But I think it doesn't hurt!